carbon.conf.example

```diff
@@ -0,0 +1,483 @@
+[cache]
```

Comment:
Has this file changed from the default example file? Not sure why it's being copied in again?